packages/component-library/src/components/plmg-dropdown-item/test/plmg-dropdown-item.spec.tsx

```diff
@@ -0,0 +1,21 @@
+import { newSpecPage } from '@stencil/core/testing';
+import { PlmgDropdownItem } from '../plmg-dropdown-item';
+
+describe('plmg-dropdown-item', () => {
+  it('renders', async () => {
+    const page = await newSpecPage({
+      components: [PlmgDropdownItem],
+      html: `<plmg-dropdown-item></plmg-dropdown-item>`,
```

Comment:
Please add the essential props to make sure they are correctly rendered in the HTML (href, rel, target, text, icon)

Reply:
done